network_observability/network-observability-overview.adoc

```diff
@@ -37,7 +37,7 @@ In *Observe* -> *Dashboards*, the *Netobserv* dashboard provides a quick overvie
  * *Top byte rates received per source and destination workloads*
 
 *Infrastructure* and *Application* metrics are shown in a split-view for namespace and workloads.
-You can configure the `FlowCollector` `spec.processor.metrics` to add or remove metrics by changing the `ignoreTags` list. For more information about available tags, see the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
+You can configure the `FlowCollector` custom resource `spec.processor.metrics.includeList` to add or remove metrics. For more information about available tags, see the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference]
```

Comment:
Will the available list [items](https://github.com/netobserv/network-observability-operator/blob/main/docs/Metrics.md) be shown in the API reference? 🤔 Wondering if we should dedicate a section to documenting these list items

Reply:
I would like to see that in the downstream docs personally! I think we should make it as easy as possible for customers to configure this metric since it's probably one of the most important ones in terms of what traffic is getting observed

Reply:
Cool, I was thinking the same. It could be helpful for them to have outside the API reference! Will doc it!